runtime/src/launch/pbs-gasnetrun_ibv/launch-pbs-gasnetrun_ibv.c

```diff
@@ -69,7 +70,7 @@ static qsubVersion determineQsubVersion(void) {
 
   if (strstr(version, "NCCS")) {
     return nccs;
-  } else if (strstr(version, "PBSPro")) {
+  } else if (strstr(version, "pbs_version") || strstr(version, "PBSPro")) {
```

Comment:
I'm not sure if it's worthwhile, but if we want to maintain compatibly with the old mpp style syntax, I'd detect `pbs_version` as `pbspro_select` or the older one as `pbspro_mpp` and specialize for them

Reply:
Fixed.